.eslintrc.json

```diff
@@ -48,45 +46,36 @@
             { "selector": "property", "format": null }
         ],
 
-        "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
-        "@typescript-eslint/consistent-type-assertions": ["error", { "assertionStyle": "as" }],
-
-        "max-statements-per-line": ["error", { "max": 1 }],
-
-        "no-duplicate-imports": "off",
-        "@typescript-eslint/no-duplicate-imports": "error",
-
-        "@typescript-eslint/no-inferrable-types": "error",
-        "@typescript-eslint/no-misused-new": "error",
-        "@typescript-eslint/no-this-alias": "error",
-
-        "no-unused-expressions": "off",
-        "@typescript-eslint/no-unused-expressions": ["error", { "allowTernary": true }],
-
-        "@typescript-eslint/prefer-for-of": "error",
-        "@typescript-eslint/prefer-function-type": "error",
-        "@typescript-eslint/prefer-namespace-keyword": "error",
-        "@typescript-eslint/prefer-as-const": "error",
-
-        "quotes": "off",
+        "@typescript-eslint/brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
+        "@typescript-eslint/no-extra-semi": "error",
         "@typescript-eslint/quotes": ["error", "double", { "avoidEscape": true, "allowTemplateLiterals": true }],
-
-        "semi": "off",
         "@typescript-eslint/semi": "error",
-        "@typescript-eslint/no-extra-semi": "error",
-
-        "space-before-function-paren": "off",
         "@typescript-eslint/space-before-function-paren": ["error", {
             "asyncArrow": "always",
             "anonymous": "always",
             "named": "never"
         }],
 
-        "@typescript-eslint/triple-slash-reference": "error",
-        "@typescript-eslint/type-annotation-spacing": "error",
-        "@typescript-eslint/unified-signatures": "error",
-
-        "@typescript-eslint/no-extra-non-null-assertion": "error",
+        // Todo: enable these soon ✨
```

Comment:
Are you thinking we're going to enable _all_ of these? I think a some of them are just never going to come on, e.g.:

- `prefer-rest-params` - probably fine
- `prefer-spread` - probably fine
- `@typescript-eslint/ban-types -` fires 1153 times, gets `Symbol` totally wrong, should really should allow `{}` (especially after 4.9 made the type very useful)
- `@typescript-eslint/no-empty-function` - not personally a fan but there are only a few. mainly, this is very annoying when defining a function and you already have an error because you haven't typed the body yet
- `@typescript-eslint/no-empty-interface` - similarly, annoying when you start writing an interface, sometimes wrong when introducing a constrained generic
- `@typescript-eslint/no-explicit-any` - I don't see us doing this, realistically, fires 529 times
- `@typescript-eslint/no-unused-vars` - yes please

I feel like I'd probably just leave all of these disabled ones in a group, and then we can remove them later on a per-case basis.

Reply:
Hmm, yes, the phrasing here seems off. Changed to _`// Todo: For each of these, investigate whether we want to enable them ✨`_ - does that feel right to you?

Reply:
Oh, and from https://github.com/microsoft/TypeScript/pull/54693#discussion_r1263133096, adding in `"eslint:recommended"` prompted me to add a few more rules to this list.

Reply:
Seems better now, yeah. We can triage later.